sw/airborne/math/pprz_algebra_float.h

```diff
@@ -102,13 +102,17 @@ struct FloatRates {
   }
 
 /*
- * Returns the log of v in base of n
+ * Returns the real part of the log of v in base of n
  */
 static inline float float_log_n(float v, float n)
 {
-  if (v == 0.0) {
+  if (fabsf(v) < 1e-4) { // avoid inf
     return - 1.0E+30;
-  } else {
+  }
+  else if (fabsf(n) < 1e-4) { // avoid nan
+    return 0;
+  }
+  else {
```

Comment:
This doesn't look well formatted, can you run fix code style? Also, you don't need the else if or else here as it is implicit with the return statements.